### ISSUE TITLE
SqliteForeignKeyIndexConvention not working for Composite Primary Keys

### ISSUE THREAD

**Issue by joker-3** [msallin/SQLiteCodeFirst#109]
1. Define a class with a composite primary key
2. Add 2 many-to-many relationships to the class
3. During creation, the indices for the navigation table are not properly changed to include the table name.

Two Intermediate Tables are created:  FooCompositeKeyFooRelationshipA and FooCompositeKeyFooRelationshipB  both tables contain a IX_FooCompositeKey_Id_FooCompositeKey_Version index for the composite primary key.
    
```
     public class FooCompositeKey
    {
        [Key, Column(Order = 1)]
        public int Id { get; set; }

        [Key, Column(Order = 2), StringLength(20)]
        public string Version { get; set; }

        [StringLength(255)]
        public string Name { get; set; }

        public virtual ICollection<FooRelationshipA> FooeyACollection { get; set; }

        public virtual ICollection<FooRelationshipB> FooeyBCollection { get; set; }
    }

    public class FooRelationshipA
    {
        public int Id { get; set; }

        [StringLength(255)]
        public string Name { get; set; }

        public virtual ICollection<FooCompositeKey> Fooey { get; set; }
    }

    public class FooRelationshipB
    {
        public int Id { get; set; }

        [StringLength(255)]
        public string Name { get; set; }

        public virtual ICollection<FooCompositeKey> Fooey { get; set; }
    }
```

**Comment by joker-3** [comment]
Fix: Update line 49 in the SqliteForeignKeyIndexConvention.Apply method from
```
string propertyName = edmProperty.Name;
```
to
```
//Check for Composite Primary Keys
string propertyName = item.Constraint.FromProperties.Count > 1 ? String.Join("_", item.Constraint.ToProperties) : edmProperty.Name;
```

**Comment by msallin** [comment]
Thank you for creating the issue, it is fixed with the version 1.5.
To support this project you can
- star the repository;
- report bugs/request features by creating new issues;
- write code and create PRs;
- donate.

https://github.com/msallin/SQLiteCodeFirst/tree/v1.5.0.21#support-the-project-
